testing/bdd_test.ts

```diff
@@ -543,6 +547,14 @@ Deno.test("it()", async (t) => {
           }),
       );
 
+      await t.step(
+        "minimum options (skip)",
+        async () =>
+          await assertMinimumOptions((fn) => {
+            assertEquals(it.skip({ name: "example", fn }), undefined);
```

Comment:
I don't understand using `assertEquals()` on methods that always return `void`. Wouldn't it be fine to just call `it.skip({ name: "example", fn })`? Ditto for the other changes in this file.

Reply:
Probably it's fine to remove them from here as they are already tested in other places.